lib/puppet-lint/monkeypatches.rb

```diff
@@ -3,7 +3,7 @@
 rescue
   # monkeypatch String#% into Ruby 1.8.7
```

Comment:
Given the lower bound is now set at 2.5, I wonder how much of this could be dropped.

Reply:
yes I've wondered about this file. I'll discuss this with the team

Reply:
hey @ekohl. I had a look at this file and am not entirely sure what its purpose is and so don't feel entirely confident to make any changes to it. In the interest of moving on with adding ruby 3 support across our tools I'm going to get this PR merged but add a ticket to our backlog to investigate this more. Thanks for highlighting this for us.